scripts/verify.sh

```diff
@@ -9,7 +9,7 @@ function launch_pylint()
 
 function launch_unit_tests()
 {
-    nose2 -v --with-coverage --coverage sfctl
+    nosetests -v --with-coverage --cover-package=sfctl --cover-inclusive
```

Comment:
Looks like your changes overrode mine - can you fix pls? Thanks :)

FYI: I discovered a couple of days ago that git rebase is really useful for cases like this.